### PR TITLE
Update docs for epoch support

### PR DESCRIPTION
**Pull request by mohabusama**

### CODE REVIEW

File: docs/user/check-commands.rst [hunk 1]
```diff
@@ -502,7 +502,7 @@ The following general-purpose functions are available in check commands:
 
     Given a time specification such as ``'-10m'`` for "ten minutes ago" or ``'+3h'`` for "in three hours", returns an object representing that timestamp. Valid units are ``s`` for seconds, ``m`` for minutes, ``h`` for hours, and ``d`` for days.
 
-    The time specification `spec` can also be a valid ISO timestamp in of the following formats:
+    The time specification `spec` can also be an ``epoch`` or a valid ISO timestamp in of the following formats:
```
Comment (hjacobs) [inline docs/user/check-commands.rst]: maybe (to make it clearer) "... can also be an Unix epoch/timestamp or ..." (see https://en.wikipedia.org/wiki/Unix_time)

Comment (mohabusama) [inline docs/user/check-commands.rst]: Done

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**